apps/trackers/jira/query.py

```diff
@@ -447,3 +477,315 @@ def query_comment(self):
         Retrieves the comment generated by .query(). Can be None if no comment was generated.
         """
         return self._comment
+
+
+class TrackerJiraQueryBuilder(OldTrackerJiraQueryBuilder):
```

Comment:
The methods that are _not_ marked with `# TODO test it with real jira` I tested with stage Jira and they worked in this manual ad-hoc testing.